.github/workflows/benchmarks.yaml

```diff
@@ -0,0 +1,44 @@
+name: Benchmarks
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+    inputs:
+      commit_ref:
+        description: "optional: compare to specific ref"
+        required: false
+
+jobs:
+  benchmarks:
+    name: Benchmark scripts
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
```

Comment:
Do you think we would gain anything (certainty about Python version, runtime stability) if we would explicitly select a Python version by using https://github.com/actions/setup-python?